package.json

```diff
@@ -39,25 +39,25 @@
     "upupdowndownleftrightleftrightbastart"
   ],
   "dependencies": {
-    "@babel/core": "^7.17.8",
+    "@babel/core": "^7.18.6",
     "@babel/eslint-plugin": "^7.17.7",
-    "@babel/plugin-proposal-decorators": "^7.16.7",
-    "@babel/plugin-proposal-export-default-from": "^7.16.7",
-    "@babel/plugin-transform-react-inline-elements": "^7.16.7",
-    "@babel/plugin-transform-runtime": "^7.16.7",
-    "@babel/preset-env": "^7.16.7",
-    "@babel/preset-react": "^7.16.7",
-    "@babel/preset-typescript": "^7.16.7",
-    "@babel/runtime": "^7.16.7",
-    "@enact/dev-utils": "^5.0.0-rc.1",
+    "@babel/plugin-proposal-decorators": "^7.18.6",
+    "@babel/plugin-proposal-export-default-from": "^7.18.6",
+    "@babel/plugin-transform-react-inline-elements": "^7.18.6",
+    "@babel/plugin-transform-runtime": "^7.18.6",
+    "@babel/preset-env": "^7.18.6",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.18.6",
+    "@babel/runtime": "^7.18.6",
+    "@enact/dev-utils": "enactjs/dev-utils#feature/WRO-6377",
     "@enact/template-sandstone": "^1.5.0-rc.1",
-    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.4",
-    "@testing-library/jest-dom": "^5.14.1",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.7",
+    "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
-    "@testing-library/user-event": "^13.2.0",
-    "@typescript-eslint/eslint-plugin": "^5.5.0",
-    "babel-jest": "^27.5.1",
-    "babel-loader": "^8.2.3",
+    "@testing-library/user-event": "^13.5.0",
+    "@typescript-eslint/eslint-plugin": "^5.30.0",
+    "babel-jest": "^28.1.2",
```

Comment:
Let's not bump up the major version. It should be aligned with jest.

Reply:
You're right, it may modified accidently.